### PR TITLE
prod ready images

### PR DESCRIPTION
**Pull request by froz42**

### CODE REVIEW

File: srcs/.env.example [hunk 1]
```diff
@@ -1,15 +1,9 @@
 DB_NAME=
 DB_USER=
-DB_PASSWORD=
+DB_NAME=
```
Comment (tsiguenz) [inline srcs/.env.example]: Duplicate

File: srcs/.env.example [hunk 2]
```diff
@@ -1,15 +1,9 @@
...
-WP_ADMIN=
-WP_ADMIN_PASSWORD=
-WP_ADMIN_MAIL=
-WP_USER=
-WP_USER_PASSWORD=
-WP_USER_MAIL=
-WP_TITLE=
-WP_URL=
+
+DOMAIN_NAME=
```
Comment (tsiguenz) [inline srcs/.env.example:6]: It's WP_URL on main, but I can change it

File: srcs/wordpress/scripts/entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        echo "Error: $2 is not set"
+        exit 1
+    fi
+}
+
+CONFIGURED_FILE=/var/www/html/.configure
+
+if [[ ! -f $CONFIGURED_FILE ]]; then
+    echo "Configuring wordpress"
+    check_env "$WP_URL" "WP_URL"
```
Comment (tsiguenz) [inline srcs/wordpress/scripts/entrypoint.sh:18]: Change to DOMAIN_NAME or change the variable in .env.template

**Review by tsiguenz — CHANGES_REQUESTED** [review]
I can't test it now but it looks good except for .env.template  :smile:

File: srcs/wordpress/scripts/entrypoint.sh (hunk 3 above)
Comment (froz42) [reply srcs/wordpress/scripts/entrypoint.sh:18]: no because WP_URL: https://${DOMAIN_NAME} is defined in env

**Comment by froz42** [comment]
Fixed

**Review by tsiguenz — APPROVED** [review]

**Pull request merged**